config/locales/en/invoice.yml

```diff
@@ -55,6 +55,7 @@ en:
     progressive_billing_credit: Usage already billed
     quarter: quarter
     quarterly: Quarterly
+    reached_usage_threshold: This progressive billing is generated because your cumulative usage has reached %{usage_amount}, exceeding the ${threshold_amount} threshold.
```

Comment:
```suggestion
    reached_usage_threshold: This progressive billing is generated because your cumulative usage has reached %{usage_amount}, exceeding the %{threshold_amount} threshold.
```